linux_docker_resources/Dockerfile

```diff
@@ -116,6 +121,19 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
   uncrustify \
   yamllint
 
+
+# Configure proposed source
+RUN if test \( ${UBUNTU_DISTRO} = noble \); then\
+    echo "deb http://archive.ubuntu.com/ubuntu/ "$(lsb_release -cs)"-proposed restricted main multiverse universe" >> /etc/apt/sources.list.d/ubuntu-$(lsb_release -cs)-proposed.list \
+;fi
+# Install from proposed conflicting packages
```

Comment:
What does `conflicting` mean in this context?

Reply:
It meant to mean packages that are broken/outdated on the repository and it needs to install from proposed instead. 
But I can see why `conflicting` can be confusing, does something like `necessary` or `broken` better?